### PR TITLE
Added a new variable and updated documentation

### PR DESCRIPTION
**Pull request by jmdaweb**
This pull request includes the following changes:

* Documentation: all http links are now https.
* Pip is used instead of easy_install.
* Added a new variable to buildVars, baseLanguage. Useful when primary documentation, source code and interface aren't written in english. This variable is taken into account when copying the readme file located in the root directory to the doc folder.

### CODE REVIEW

**Comment by josephsl** [comment]
Hi, follow-up discussion posted to NVDA Add-ons list. If approved, this will be part of Add-on Template 2020.4 (December at the earliest) as 2020.3 edition is frozen at the moment. Thanks.

File: buildVars.py [hunk 1]
```diff
@@ -59,3 +59,8 @@ def _(arg):
 # Files that will be ignored when building the nvda-addon file
 # Paths are relative to the addon directory, not to the root directory of your addon sources.
 excludedFiles = []
+
+# Base language for the NVDA add-on
+# If the source code, add-on interface and readme file located in the root folder are written in a language other than english, modify this variable as appropriate.
```
Comment (lukaszgo1) [inline buildVars.py:64]: Please split this line up. While we don't have strict requirement about line lengths I believe 80-90 chars is a sane maximum.

**Comment by josephsl** [comment]
Hi,

Please do not host a pull request on master branch directly, as it will confuse everyone later (it'll confuse reviewers because master branch can become messy; it confuses authors because they need to keep up with changes from original master branch). I will take this in on behalf of the community after 2020.3 is released, but in the future, please use separate pull request branches in alignment with NVDA project itself.

Thanks.

**Comment by josephsl** [comment]
Hi,

Three things:

* I'll correct spelling and wording issues once merged.
* Would it be better to put 'base language' flag in the manifest? I suggest this as it may become handy for things other than documentation, and so far the only place the new flag is used is when building documentation. But then again, it may pose problems in case NVDA Core decides to use base language flag in the manifest for various reasons.
* I think it would be best to not solve HTTPS and PIP problems with this PR - something that can be done with a separate issue/PR pair. But since this PR includes a fix, I'm willing to make an exception (please don't do that again).

Other than that, I approve this PR. Comments from other people are appreciated.

Thanks.

**Comment by jmdaweb** [comment]
Hi Joseph,
Sorry, I will take it into account for the next time.
Regards.

**Pull request merged**